.github/workflows/test.yml

```diff
@@ -257,187 +255,3 @@ jobs:
           working-directory: 'tests/rextendr'
           check-dir: '"rextendr_check"'
           error-on: '"${{ steps.error-on.outputs.level }}"'
-
-  # All tests under this job are run with R devel and freshly generated bindings.
-  # Run bindgen tests without cross-compilation.
-  # Cross-compilation prevents from running doc tests
-  # if toolchain does not match target (Windows x86_64/i686 case), see
-  #  > https://github.com/rust-lang/rust/issues/64245
-  #  > https://github.com/rust-lang/cargo/issues/7040
-  bindgen:
-    runs-on: ${{ matrix.config.os }}
-
-    name: ${{ matrix.config.os }} w/ bindgen (R-${{ matrix.config.r }} rust-${{ matrix.config.rust-version }} ${{join(matrix.config.rust-targets, ',')}})
-    
-    strategy:
-      fail-fast: false
-      matrix:
-        config:
-          - { os: ubuntu-latest,  r: 'devel',   rust-version: 'stable' }
-          - { os: macOS-latest,   r: 'devel',   rust-version: 'stable' }
-          - { os: windows-latest, r: 'release', rust-version: 'stable-msvc',  rust-targets: [ 'x86_64-pc-windows-gnu' ], rtools-version: '44' }
-          - { os: windows-latest, r: 'devel',   rust-version: 'stable-msvc',  rust-targets: [ 'x86_64-pc-windows-gnu' ], rtools-version: '44' }
-          # When the MSVC target is used, since it's cross-compilation from MSVC
-          # to GNU, the doc tests are usually skipped. Adding `-Zdoctest-xcompile`
-          # lets the doctests run, which accordingly require the nightly toolchain.
-          - { os: windows-latest, r: 'release', rust-version: 'nightly-msvc', rust-targets: [ 'x86_64-pc-windows-gnu' ], rtools-version: '44', extra-args: [ '-Zdoctest-xcompile' ] }
-          - { os: windows-latest, r: 'oldrel',  rust-version: 'nightly-msvc', rust-targets: [ 'x86_64-pc-windows-gnu' ], rtools-version: '43', extra-args: [ '-Zdoctest-xcompile' ] }
```

Comment:
Note that these Rust versions `nightly-msvc` with `extra-args: [ '-Zdoctest-xcompile']` are not in the remaining CI configs. Do we want to add these? 

What benefit do we get testing on nightly over release?

Reply:
Z flags won't work on stable.

Reply:
Totally understand. So do we want to include these versions into the matrix config so that we get doc testing?

Reply:
Yes. Have you looked at the steps in both jobs?